### PR TITLE
Inflected: update type definitions to version 2.1.0

### PR DESCRIPTION
**Pull request by Jclerck**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/martinandert/inflected
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## inflected ([<kbd>unpkg</kbd>](https://unpkg.com/browse/inflected@latest/))
was missing the following properties:
1. capitalize
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against be8f753641511c573a92e3bcdffc8e3c4cbfd4ca
</p>

File: types/inflected/index.d.ts
Comment (peterblazejewicz) [inline types/inflected/index.d.ts:1]: What about simplifying this? There is no really 'Inflected' namesapce or symbol, etc.
That looks like a JS module with just functions exported (and also exported as UMD module),
so:
```ts
export interface Humanize {
    capitalize?: boolean;
}

export interface Parameterize {
    preserveCase?: boolean;
    separator?: string;
}

export interface Transliterate {
    locale?: string;
    replacement?: string;
}

export interface Inflections {
    acronym(word: string): void;
    plural(rule: RegExp, replacement: string): void;
    plural(rule: string): void;
    singular(rule: RegExp, replacement: string): void;
    singular(rule: string): void;
    irregular(singular: string, plural: string): void;
    uncountable(...words: string[]): void;
    human(rule: RegExp | string, replacement: string): void;
    clear(scope?: string): void;
}

export interface Transliterations {
    approximate(original: string, replacement: string): void;
}

export function pluralize(word: string, locale?: string): string;
export function singularize(word: string, locale?: string): string;
export function camelize(term: string, uppercaseFirstLetter?: boolean): string;
export function underscore(camelCaseWord: string): string;
export function humanize(lowerCaseAndUnderscoredWord: string, options?: Humanize): string;
export function titleize(sentence: string): string;
export function tableize(className: string): string;
export function classify(tableName: string): string;
export function dasherize(underscoredWord: string): string;
export function foreignKey(className: string, separateClassNameAndIdWithUnderscore?: boolean): string;
export function ordinal(number: number): string;
export function ordinalize(number: number): string;
export function inflections(locale: string, inflect?: (inflect: Inflections) => void): void;
export function inflections(inflect?: (inflect: Inflections) => void): void;
export function transliterate(sentence: string, options?: Transliterate): string;
export function transliterations(locale: string, transliterate?: (transliterate: Transliterations) => void): void;
export function transliterations(transliterate?: (transliterate: Transliterations) => void): void;
export function parameterize(sentence: string, options?: Parameterize): string;
export function constantify(words: string): string;
export function capitalize(word?: string | null): string;


export as namespace Inflector;
```

File: types/inflected/index.d.ts [hunk 1]
```diff
@@ -1,43 +1,59 @@
-// Type definitions for inflected 1.1.6
+// Type definitions for inflected 2.1.0
```
Comment (peterblazejewicz) [inline types/inflected/index.d.ts:1]:
```suggestion
// Type definitions for inflected 2.1
```
just MAJ.MIN:
https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@Jclerck thanks!

**Comment by Jclerck** [comment]
Ready to merge

**Pull request merged**